validation/genesis/validation-inputs/8008/meta.toml

```diff
@@ -0,0 +1,4 @@
+genesis_creation_commit = "d80c145e0acf23a49c6a6588524f57e32e33b91c"
+node_version = "18.12.1"
+monorepo_build_command = "pnpm"
+genesis_creation_command = "forge1"
```

Comment:
Let's try "opnode1" here instead of "forge1"

Reply:
made @bitwiseguy

Reply:
There are >2500 line diffs when it tries to compare the genesis.allocs that are being uploaded as part of this pr versus what it is able to regenerate from the optimism monorepo commit hash provided in this file. This makes me think the commit hash that was actually used to create the original genesis file is not `d80c145e0acf23a49c6a6588524f57e32e33b91c`. Do you have any idea what it might be if not that one?

One major diff is the uploaded genesis.allocs contains entries for addresses `0x00-0xff` all with `"balance": "0x1"`,

Reply:
used the tooling commit and opnode2. validation passed